src/main/java/org/zalando/nakadi/domain/NakadiCursor.java

```diff
@@ -79,15 +80,18 @@ public boolean equals(final Object o) {
                 && Objects.equals(this.partition, that.partition)
                 && Objects.equals(this.offset, that.offset);
     }
-
-    @Override
-    public int compareTo(final NakadiCursor other) {
-        final int orderDiffers = Integer.compare(this.getTimeline().getOrder(), other.getTimeline().getOrder());
-        if (0 != orderDiffers) {
-            return orderDiffers;
-        }
-        return this.getOffset().compareTo(other.getOffset());
-    }
+//
```

Comment:
Why is this commented out?

Reply:
Is it because you want to move the comparison to another class? Maybe just ✂️  this

Reply:
@rcillo Yep, I just forgot to remove it, will do.